.github/actions/setup-deps/action.yaml

```diff
@@ -111,6 +113,7 @@ runs:
         CONDA_OPT_DEPS: |
           ${{ inputs.chemfiles-python }}
           ${{ inputs.clustalw }}
+          ${{ inputs.distopia }}
```

Comment:
As per my previous comment, can we not do this please? Distopia is already tested upstream so we'd want to keep testing our full deps CI to be what's in core's lib for now. Please just add a separate runner with the extra conda dep defined

Reply:
Sorry misinterpreted your CI comment.

Reply:
@IAlibay sorry I am a little confused. Is this not in the right spot under `CONDA_OPT_DEPS`?

Reply:
Can I query this again @IAlibay? If it needs to go somewhere else I'll move it, but I followed the same process in #3765. Perhaps I need to change both.

Reply:
Sorry I know there's a few things in my review pile, I'll try to deal with them over the next few days (i.e. before the 2.4.0 deadline)

Reply:
Because it's not an optional backend and you're not preventing non-distopia things to run at the same time, this isn't a problem anymore. If it wasn't a non-optional backend, then there's a specific way to create matrix additions and pass extra things that need to get installed to the "extra-conda-deps" input (which is what I was referring to).